lib/connectors/memory.js

```diff
@@ -788,7 +855,9 @@ Memory.prototype.replaceById = function(model, id, data, options, cb) {
   var self = this;
   if (!id) {
     var err = new Error(g.f('You must provide an {{id}} when replacing!'));
-    return process.nextTick(function() { cb(err); });
+    return process.nextTick(function() {
```

Comment:
`process.nextTick(cb, err)` while you are at changing this (why is the change needed in the first place?)